lib/senkyoshi/canvas_course.rb

```diff
@@ -161,9 +189,11 @@ def upload_scorm_package(scorm_package, course_id, tmp_name)
           },
           Authorization: "Bearer #{AuthToken.issue_token}",
         ) do |resp|
-          result = JSON.parse(resp.body)["response"]
-          result["points_possible"] = scorm_package.points_possible
-          result
+          result = JSON.parse(resp.body)
+          response = result["response"]
```

Comment:
This is more descriptive of the process, but is there any other reason we are making `result` it's own variable if it's never used except to get the response out of it?

Reply:
It is used on line 195.

Reply:
You are correct, my apologies!